mentor-notes.md

```diff
@@ -0,0 +1,22 @@
+## Notes for mentors
+
+### Structure of Software Design Workshop
+
+This is a 3 hour workshop with probably more material and exercises to complete than is likely possible to. As such, mentors should let students know in advance that they probably will not complete everything and instead should focus on understanding the material.
+
+Mentors can help students achieve this by telling them to spend around 15 minutes reading each section in Stage 1. After this mentors should spend 10 minutes talking through the material and lollipop questions (see some examples below). After this, students can either move on to the next section or on to the exercise if the section has one.
+
+### Questions
+
+**First-class functions:**
+* What is a first-class function?
+* Can you give two examples of how functions can be used?
+
+**Abstraction with functions**
+* Can you give an example of when you might use abstraction?
```

Comment:
It may or may not be useful at this point to look at how some of the tools Javascript provides natively can be seen as abstractions. For example, you could look at some of the array methods (`map`, `sort`, `find`, etc) and discuss how they ultimately all abstract the process of iteration in some way. Similarly, looking at the methods provided by the `Element.classList` object, which essentially provide abstraction over string manipulation of DOM elements. I'm sure you could think of other examples too.

Anyway, just a thought.

Reply:
This is a great shout, thanks Elias!

Reply:
@eliascodes i think this is a good idea.
@ameliejyc do you and @rebecacalvoquintero have intention of making that change?

@eliascodes does your approval of this PR depend on that change, or are you happy for it to go in as is?

Reply:
Nah, I'm good, I'll merge now.

Reply:
@mantagen yes of course...we will think about other examples :) I think @eliascodes made a good point :) <3